language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/providers/context/FromClauseNodeContext.java

```diff
@@ -123,13 +123,24 @@ public void sort(BallerinaCompletionContext context,
                 TypeDescKind.STRING, TypeDescKind.ARRAY,
                 TypeDescKind.MAP, TypeDescKind.TABLE,
                 TypeDescKind.STREAM, TypeDescKind.XML);
-        
+
```

Comment:
IMHO, we can get rid of this lengthy logic by simply using `SortingUtil.genSortTextByAssignability()`. WDYT? @LakshanWeerasinghe

Reply:
@malinthar do you mean the logic inside the `foreach`?

Reply:
Yes. Just like we have done here 
https://github.com/ballerina-platform/ballerina-lang/blob/f5c0c7bc2a469940a40c577c5a3d5068b74e4207/language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/providers/context/AssignmentStatementNodeContext.java#L119-L133

Reply:
@malinthar we cannot use this for some scenarios the typeSymbolAtCursor` can be empty. ex: `from var item in <cursor>`.

But in here we can still need to prioritize iterables.

Reply:
Agreed. Let's simplify the logic as much as we can